fl4health/utils/config.py

```diff
@@ -4,7 +4,6 @@
 
 REQUIRED_CONFIG = {
     "n_server_rounds": int,
-    "n_clients": int,
```

Comment:
I remove this as a required config param because I wanted to have the number of clients to be dynamic and passed in the launching script. If we put it in a config, then we cannot (easily) check the value of n_clients from the launch script to dynamically launch the given number of clients.

Reply:
Makes sense to me!